03/index.html

```diff
@@ -5,12 +5,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #03</title>
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(min-width: 600px, max-width: 1199px)" href="./styles/tablet.css">
+    <link rel="stylesheet" media="(min-width: 1200px)" href="./styles/desktop.css">
 </head>
 <body>
     
-    <main class="t">T</main>
-    <header class="p">P</header>
-    <side class="v">V</side>
+    <main class="t">Lorem ipsum dolor sit amet consectetur adipisicing elit. Est qui pariatur ullam earum non. Exercitationem quidem distinctio, maxime voluptatum ea esse mollitia debitis autem nemo blanditiis, possimus quibusdam. Cum, corrupti. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Earum delectus, deserunt ipsa consequuntur aperiam quasi maxime saepe excepturi at iusto, aliquam nesciunt sunt odio dignissimos fugiat necessitatibus illum corporis sint. Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid labore ea repudiandae repellat laborum. Ab perspiciatis aliquam assumenda magni officiis minus ullam cupiditate cumque, culpa sapiente molestiae, voluptates quidem nemo!</main>
+    <header>
```

Comment:
Jeśli ma działać "grid" to klasa `p` powinna być przypisana do `header` (grid działa na dzieciach, a nie wnukach).